src/blog/_posts/2023-08-01-introduction-to-cadence-signals.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Introduction to Cadence Signals
+
+date: 2023-08-06
+author: Chris Qin
+authorlink: https://www.linkedin.com/in/chrisqin0610/
+---
+
+In this blog, we will have a brief overview of Cadence signals, a critical feature that provides a way to directly send data to a running workflow. 
+
+In previous blogs, we have gone through on how to implement a simple HelloWorld workflow, which takes a single string as workflow input and executes another activity to obtain the output. In that, you’ve learned two options to pass data to a workflow
```

Comment:
In previous blogs, we have covered how to....

In that example, you learned about two options of how to pass data to a workflow